src/AwsServiceProvider.php

```diff
@@ -22,13 +22,13 @@ class AwsServiceProvider extends ServiceProvider {
      */
     public function boot()
     {
-        $config = realpath(__DIR__ . '/../config/config.php');
+        $source = realpath(__DIR__ . '/../config/aws.php');
 
-        $this->mergeConfigFrom($config, 'aws');
+        if (class_exists('Illuminate\Foundation\Application', false)) {
```

Comment:
Note that for efficiency reasons I've told php not to bother trying to autoload the class. If it's not already loaded by the time this service provider is booted, then we can assume we're not running Laravel.